### DISCUSSION TITLE
Help with db migration

### DISCUSSION THREAD

**Discussion by frimen** [outline/outline#4100] [Show and tell]
Hello everyone. Please help me, I don't know how to do migration. I do as it says in the instructions, it gives an error.

root@host:/# docker-compose run --rm outline yarn outline:migrate --env=production-ssl-disabled
ERROR: No such service: outline

**Comment by frimen** [comment]
Please delete the topic. The issue is resolved.

**Reply by tommoor** [reply]
Please leave your solution for future community members

**Reply by CHUYKOPS** [reply]
You need to change command line in docker compose like:

command: sh -c "yarn db:migrate --env=production-ssl-disabled && yarn start --env=production-ssl-disabled"
